common/types/mapper/proto/helpers.go

```diff
@@ -79,6 +79,15 @@ func timeToUnixNano(t *gogo.Timestamp) *int64 {
 	return common.Int64Ptr(timestamp.UnixNano())
 }
 
+func timeStampToTime(t *gogo.Timestamp) *time.Time {
```

Comment:
I suggest writing helpers for both conversion in both directions. Both thrift and proto.

Reply:
wrote helper for thrift + added test for said helper.